docfx/en/components/toc.json

```diff
@@ -63,34 +63,28 @@
     "href": "general-licensing.md",
     "status": ""
   },
+  {
+    "exclude": ["Blazor", "React", "WebComponents"],
+    "name": "Changelog",
+    "href": "general-changelog-dv.md",
+    "status": "UPDATED"
+  },
   {
     "exclude": ["Angular", "React", "WebComponents"],
-    "name": "How to",
-    "status": "NEW",
-    "items": [
-      {
-        "exclude": ["Angular", "React", "WebComponents"],
-        "name": "Blazor Excel Library",
-        "href": "blazor-excel-library-temporarily-stop-automatic-calculation-of-formulas-to-speed-up-processing.md",
-        "status": ""
-      },
-      {
-        "exclude": ["Angular", "React", "WebComponents"],
-        "name": "How To Read & Write Excel Files To Reduce Server Load",
-        "href": "blazor-webassembly-how-to-read-and-write-excel-files-to-reduce-server-load.md",
-        "status": ""
-      },
-      {
-        "exclude": ["Angular", "React", "WebComponents", "Blazor"],
-        "name": "How To Customize Marker Display in Blazor Charts",
-        "href": "customize-marker-display-in-blazor-charts.md",
-        "status": ""
-      }
-    ]
+    "name": "Changelog",
+    "href": "general-changelog-dv-blazor.md",
+    "status": "UPDATED"
   },
   {
+    "exclude": ["Angular", "Blazor", "WebComponents"],
     "name": "Changelog",
-    "href": "general-changelog-dv.md",
+    "href": "general-changelog-dv-react.md",
+    "status": "UPDATED"
+  },
+  {
+    "exclude": ["Angular", "React", "Blazor"],
+    "name": "Changelog",
+    "href": "general-changelog-dv-wc.md",
     "status": "UPDATED"
```

Comment:
move these 3 nodes above the node for "general-changelog-dv.md" (Angular)

![image](https://user-images.githubusercontent.com/11665978/196534047-74c5ca30-e29f-4474-a9c9-0cf5253c06d4.png)

Reply:
see 
https://github.com/IgniteUI/igniteui-xplat-docs/pull/383/commits/6d6fce67106c04140fb7dae241e45beadabc6d62